backend/Agent/research_paper/server/healthcare_v2_en.py

```
@@ -1156,6 +1162,422 @@ async def check_emergency_keywords(context: ToolContext, text: str) -> ToolResul
+            if night_dialysis:
+                filters["night_dialysis"] = True
+
+            results = await HOSPITAL_MANAGER.search_by_text(
+                query=query,
+                limit=max_results,
+                filters=filters if filters else None
+            )
+
+        else:
+            # Default: Get dialysis centers
+            logger.info(f"[HOSPITAL TOOL] Using default: get dialysis centers")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)